### PR TITLE
COE-513: OEE release notes

### PR DESCRIPTION
**Pull request by mvoigt-sag**

### CODE REVIEW

**Comment by BeateRixen** [comment]
@mvoigt-sag @thomaswinkler I did the language review and edits directly in the source file. Please note that these release notes might not be sufficient for some customers. We usually provide much more infos for each entry, at least always describing the context, the change and the impact for the user. Compare also https://docs.google.com/document/d/1SnU3xMFhBbAebr7r6oxtLjx9NY9WFCoytyESV2xyQw8/edit

**Review by BeateRixen — APPROVED** [review]

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+<td style="text-align:left">COE-416</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Changed the calculation of availability to not require status changes and always create a measurement at the end of an interval.</td>
+<td style="text-align:left">COE-335</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Only 1 measurement with 20 fragments per calculation interval will be stored now instead of 20 measurements. This reduces the number of MEAs created by the OEE app significantly.</td>
```
Comment (haraldmeyer) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:65]: OEE app -> Cumulocity IOT OEE App

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+<td style="text-align:left">COE-387</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Fixed the accuracy and rounding of the quality calculation that might have caused sending additional alarms.</td>
+<td style="text-align:left">COE-351</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Deactivated the CSV handler.</td>
```
Comment (haraldmeyer) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:75]: Do we need  to mention this here? It was not accessible anyway.

**Review by haraldmeyer — CHANGES_REQUESTED** [review]
Two minor comments.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 2 above)
Comment (thomaswinkler) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:75]: Agree. Will remove it.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 1 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:65]: Implemented the fix

**Comment by thomaswinkler** [comment]
@BeateRixen @MWindel @mvoigt-sag @haraldmeyer 

Pushed a new changed going over the release notes again to provide more information as requested.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 2 above)
Comment (thomaswinkler) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md]: Removed.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 3]
```diff
@@ -26,72 +26,70 @@ Most important change to highlight is storing of OEE values in just 1 measuremen
...
 <tr>
 <td style="text-align:left">UI</td>
-<td style="text-align:left">Fix trend arrow calculation and presentation in the organisation overview.</td>
+<td style="text-align:left">The OEE targets in the overview do show trend arrows next to the OEE value indicating if the OEE trend is going up or down. An issue has been fixed breaking the correct calculation of the trend, so even a trend arrow is presented it might not have represented the calculated trend. Also a new trend array (right) has been introduced to indicate trends without a change.</td>
 <td style="text-align:left">COE-416</td>
 </tr>
 <tr>
 <td style="text-align:left">Microservice</td>
-<td style="text-align:left">Fix calculation of Availability to not require status changes and always create measurement at the end of an interval.</td>
+<td style="text-align:left">The Availability is now always calculated at the end of each interval without relying on status change events, such as machine up or down events. Depending on the status changes detected for machines, it was possible to not see any OEE being calculated for multiple calculation intervals or as expected at the end of each calculation interval.</td>
```
Comment (haraldmeyer) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:59]: @thomaswinkler This is not strictly correct. The trigger is still an MEA from the formula that arrives after the interval has ended to trigger a recalculation.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 4]
```diff
@@ -26,72 +26,70 @@ Most important change to highlight is storing of OEE values in just 1 measuremen
...
 <td style="text-align:left">COE-335</td>
 </tr>
 <tr>
 <td style="text-align:left">Microservice</td>
-<td style="text-align:left">Store only 1 measurement with 20 fragments per calculation interval. This does reduce the number of MEAs created by OEE app significantly.</td>
+<td style="text-align:left">Store only 1 measurement with <code>type="OEEMeasurements"</code> including all calculated OEE values and parameters as separate fragments for each calculation interval. This reduces the number of measurements created per calculation interval from 20 to just 1. OEE measurement queries should be updated to only use <code>type="OEEMeasurements"</code> instead of quering for <code>types</code> or <code>valueFragmentType</code> separately.
+</td>
```
Comment (haraldmeyer) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:65]: @thomaswinkler Will old data that is spread across multiple measurements still be shown in the OEE App? I think we should mention this.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 5]
```diff
@@ -26,72 +26,70 @@ Most important change to highlight is storing of OEE values in just 1 measuremen
...
 <td style="text-align:left">COE-387</td>
 </tr>
 <tr>
 <td style="text-align:left">Microservice</td>
-<td style="text-align:left">Fix accuracy and rounding of Quality calculation that might have caused sending additional alarms.</td>
+<td style="text-align:left">The accuracy and rounding of Quality calculation has been fixed. Even for an expected Quality of 100.0, the calculated Quality value might have slighlty deviated. The Quality calculation  has been fixed and no more Quality accuracy related OEECalculationAlarms will be triggered for the calculated interval.
```
Comment (haraldmeyer) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:70]: Spelling error in "slightly". Also to make more clear: "In the past, the calculated Quality value might have slightly deviated from the expected value causing a Quality value in excess of 100.0. and an OEECalculationAlarm"

**Review by haraldmeyer — CHANGES_REQUESTED** [review]
See detailed comments.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 4 above)
Comment (mvoigt-sag) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:65]: @thomaswinkler I think the compatibility with old data (i.e. 20 measurements) is the reason why we didnt optimize the UI code (yet) - as we discussed yesterday. If we'd change the REST requests in oee-ui, you wouldnt see the OEE measurements that were created before the fix. I think this change in oee-ui should be handled in Fix4 or later.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+<th style="text-align:left">Component</th>
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">Enabled custom tenant UI branding. OEE App now automatically uses the tenants Branding configuration to enable customization of colors, fonts, logos etc. For more information on Cumulocity IoT Branding configuration see ...</td>
```
Comment (mvoigt-sag) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:29]: Maybe the link to the doc is missing here to replace the trailing three dots?

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+<td style="text-align:left">COE-350</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">When creating a new OEE calculation profile, a list of existing calculation profiles is presented to optionally choose copy an existing configuration into the new calculation profile. An issue has been fixed causing the list of existing calculation profiles to not show in some situations.</td>
```
Comment (mvoigt-sag) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:34]:
```suggestion
<td style="text-align:left">When creating a new OEE calculation profile, a list of existing calculation profiles is presented to optionally choose to copy an existing configuration into the new calculation profile. An issue has been fixed causing the list of existing calculation profiles to not show in some situations.</td>
```

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Store only 1 measurement with <code>type="OEEMeasurements"</code> including all calculated OEE values and parameters as separate fragments for each calculation interval. This reduces the number of measurements created per calculation interval from 20 to just 1. OEE measurement queries should be updated to only use <code>type="OEEMeasurements"</code> instead of quering for <code>types</code> or <code>valueFragmentType</code> separately.
+</td>
+<td style="text-align:left">COE-387</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">The accuracy and rounding of Quality calculation has been fixed. Even for an expected Quality of 100.0, the calculated Quality value might have slighlty deviated. The Quality calculation  has been fixed and no more Quality accuracy related OEECalculationAlarms will be triggered for the calculated interval.
```
Comment (mvoigt-sag) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:70]:
```suggestion
<td style="text-align:left">The accuracy and rounding of Quality calculation has been fixed. Even for an expected Quality of 100.0, the calculated Quality value might have slighlty deviated. The Quality calculation has been fixed and no more Quality accuracy related OEECalculationAlarms will be triggered for the calculated interval.
```

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+</td>
+<td style="text-align:left">COE-351</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Improved and optimized logging of HttpErrors logged by the GeneralErrorHandler in logs of oee-apama microservice.</td>
```
Comment (mvoigt-sag) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:76]:
```suggestion
<td style="text-align:left">Improved and optimized logging of HttpErrors logged by the GeneralErrorHandler in logs of apama-oeeapp microservice.</td>
```

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 5 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md]: I implemented the changes

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 6 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 3 above)
Comment (haraldmeyer) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:61]: @MWindel "The availability is now always calculated upon the arrival of the next MEA after the end of each interval without relying on a status change, such as machine goes down, happening. Depending on the status changes detected for machines, it was possible to not see any OEE being calculated for long periods of time."

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 4 above)
Comment (haraldmeyer) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:67]: @MWindel I suggest to add one sentence at the end: "The updated OEE App is backwards-compatible and will also show data stored in separate measurements as it was stored before the change."

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 3 above)
Comment (thomaswinkler) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md]: Did push changes trying to address @haraldmeyer comment.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 4 above)
Comment (thomaswinkler) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:67]: Did push changes trying to address comments.

Comment (mvoigt-sag) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:67]: @thomaswinkler sounds good to me!

**Comment by MWindel** [comment]
@haraldmeyer For this PR to be merged, you need to remove your "request for changes". If the content is good, please approve, so this PR can be merged.

**Review by thomaswinkler — APPROVED** [review]

**Comment by BeateRixen** [comment]
@thomaswinkler @haraldmeyer I assume that this can finally be merged?

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+<th style="text-align:left">Component</th>
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">Enabled custom tenant UI branding. OEE App now automatically uses the tenants Branding configuration to enable customization of colors, fonts, logos etc. For more information on Cumulocity IoT branding configuration, see
```
Comment (BeateRixen) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:29]: the tenants Branding configuration  > the tenant´s branding configuration 
remove one instance of "for more information"

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+<a href="/users-guide/enterprise-tenant/#branding">Enterprise tenant > Customizing your platform > Branding</a> in the <i>User guide </i> for more information.
```
Comment (BeateRixen) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:30]: <i>User guide </i> > <i>User guide</i>

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+<td style="text-align:left">COE-321</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">OEE App is sending OEERestartInformation alarms when the calculation is started or stopped. An issue has been fixed sending OEERestartInformation alarms also when creating or saving a calculation profile.</td>
+<td style="text-align:left">COE-352</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">In Safari browsers, the gauge charts in OEE status of machine dashboards and Andon boards have been fixed to render correctly.</td>
```
Comment (BeateRixen) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:46]: in OEE status > in the OEE status

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+<td style="text-align:left">COE-363</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">Gauge charts in OEE status of machine dashboards and Andon boards have been fixed to render correctly for values greater than 100.</td>
```
Comment (BeateRixen) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:51]: in OEE status > in the OEE status

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
+<td style="text-align:left">COE-376</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">The OEE targets in the overview do show trend arrows next to the OEE value indicating if the OEE trend is going up or down. An issue has been fixed breaking the correct calculation of the trend, so even a trend arrow is presented it might not have represented the calculated trend. Also a new trend array (right) has been introduced to indicate trends without a change.</td>
```
Comment (BeateRixen) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:56]: do show > show

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Store only 1 measurement with <code>type="OEEMeasurements"</code> including all calculated OEE values and parameters as separate fragments for each calculation interval. This reduces the number of measurements created per calculation interval from 20 to just 1. OEE measurement queries should be updated to use <code>type="OEEMeasurements"</code> instead of quering for <code>types</code> or <code>valueFragmentType</code>. The OEE application itself supports both variants of measurements and will continue to work as expected. 
+</td>
+<td style="text-align:left">COE-387</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">The accuracy and rounding of Quality calculation has been fixed. Even for an expected Quality of 100.0, the calculated Quality value might have slightly deviated. In the past, the calculated Quality value might have slightly deviated from the expected value causing a Quality value in excess of 100.0. and an OEECalculationAlarm
```
Comment (BeateRixen) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:72]: Quality calculation > the Quality calculation 
@MWindel I can´t remember if we decided to have lowercase with "Quality"?
add full stop at the end

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+</td>
+<td style="text-align:left">COE-351</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Improved and optimized logging of HttpErrors logged by the GeneralErrorHandler in logs of apama-oeeapp microservice.</td>
```
Comment (BeateRixen) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:78]: in logs of apama-oeeapp microservice > in logs of the apama-oeeapp microservice

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 17]
```diff
@@ -0,0 +1,98 @@
...
+<td style="text-align:left">COE-420</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Improve and fixed REST API documentation for oee-bundle microservice. Documentation is now available as OpenAPI at <a href="https://cumulocity.com/oee/api/10.10.0/">https://cumulocity.com/oee/api/10.10.0/</a></td>
```
Comment (BeateRixen) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:83]: Improve > Improved
REST API documentation for oee-bundle microservice > the REST API documentation for the oee-bundle microservice

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 18]
```diff
@@ -0,0 +1,98 @@
...
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Fix REST API error handling and error messages. Error responses might have contained not required information or could have used different formats.
+</td>
+<td style="text-align:left">COE-268</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Fix and improve performance and scalability of microservices. Main fixes are related to the oee-bundle microservice.</td>
```
Comment (BeateRixen) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:94]: Fix and improve performance> Fixed and improved the performance
Main fixes > The main fixes

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 19]
```diff
@@ -0,0 +1,98 @@
...
+<td style="text-align:left">COE-350</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">When creating a new OEE calculation profile, a list of existing calculation profiles is presented to optionally choose to copy an existing configuration into the new calculation profile. An issue has been fixed causing the list of existing calculation profiles to not show in some situations.</td>
+<td style="text-align:left">COE-321</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">OEE App is sending OEERestartInformation alarms when the calculation is started or stopped. An issue has been fixed sending OEERestartInformation alarms also when creating or saving a calculation profile.</td>
```
Comment (BeateRixen) [inline content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:41]: OEE App > The OEE App

**Comment by BeateRixen** [comment]
@MWindel I added a number of language related comments, please check.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 20]
```diff
@@ -0,0 +1,98 @@
+---
+weight: 39
+title: Release 10.10.0.1
+layout: redirect
+---
+
+This release of Cumulocity IoT OEE App fixes important calculation issues and brings a set of UI improvements and enhancements for icons, dashboards, charts and profile editor.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:7]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+
+Important, all OEE values and parameters calculated per calculation interval are now stored as fragments in 1 single measurement instead of 20 measurements. With this change, the number of measurements created by calculation interval is significantly reduced.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:9]: I formatted it as an **Important** note.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 10 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:29]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 11 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:30]: @BeateRixen What does this comment refer to exactly?

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 19 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:41]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 12 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:46]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 13 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:51]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 14 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:56]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 16 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:78]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 17 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:83]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 22]
```diff
@@ -0,0 +1,98 @@
...
+<td style="text-align:left">COE-420</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Improve and fixed REST API documentation for oee-bundle microservice. Documentation is now available as OpenAPI at <a href="https://cumulocity.com/oee/api/10.10.0/">https://cumulocity.com/oee/api/10.10.0/</a></td>
+<td style="text-align:left">COE-195</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Fix REST API error handling and error messages. Error responses might have contained not required information or could have used different formats.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:88]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 11 above)
Comment (BeateRixen) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:30]: There is an empty space before the bracket, sorry, didn´t relaize that this gets reformatted in the comment:
"User guide </" > "User guide<"

Comment (BeateRixen) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:30]: There is an empty space before the bracket, sorry, didn´t relaize that this gets reformatted in the comment:
"User guide </" > "User guide<"

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md [hunk 23]
```diff
@@ -0,0 +1,98 @@
...
+<td style="text-align:left">COE-416</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">The Availability for an interval is now always calculated with the first measurement, event or alarm (MEA) after the interval ended and does not require explicit status change events, such as machine up or down events. Depending on the status changes reveived for machines, it was possible to not see any OEE being calculated at the end of each calculation interval or even for multiple intervals.</td>
+<td style="text-align:left">COE-335</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Store only 1 measurement with <code>type="OEEMeasurements"</code> including all calculated OEE values and parameters as separate fragments for each calculation interval. This reduces the number of measurements created per calculation interval from 20 to just 1. OEE measurement queries should be updated to use <code>type="OEEMeasurements"</code> instead of quering for <code>types</code> or <code>valueFragmentType</code>. The OEE application itself supports both variants of measurements and will continue to work as expected.
```
[the remark this answers is not in the document]
Comment (rssag) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:66]: i'd say "1" is fine in this case

[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:66]: Ok, then let´s leave it as is.

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 11 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:30]: Done

File: content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md (hunk 18 above)
Comment (MWindel) [reply content/release-10-10-0/oee-10-10-0-bundle/10_10_0_1.md:94]: Done

**Pull request merged**